helm/hpcc/templates/eclagent.yaml

```diff
@@ -78,11 +79,11 @@ data:
 {{ include "hpcc.addImageAttrs" (dict "root" $ "me" .) | indent 12 }}
             resources:
               requests:
-                #request-memory
-                #request-cpu
+                #requests-memory
+                #requests-cpu
```

Comment:
It maps directly to the resource structure in K8s. 
#resources:
#  requests:
#    memory: "1G"
..

It makes my code simpler.  I can pass the category name + limit name and the code can (1) lookup the value for \ChildContainerResources\[category-name]\[limit-name]" and (2)
search  "#[category-name]-[limit-name]" and replace with value.

Reply:
>It makes my code simpler. I can pass the category name 

I don't think "requests-memory" vs "request-memory" is a big deal here, 
but should aim for being as clear as possible to the end-user of these settings, rather than for making code slightly simpler.

Reply:
see other comments - issue become moot, if generating whole section.
The workunit properties that match the process and conform to the pattern of our choosing would all be picked up and used to override the generated 'resources' section.
It might be better to use a more generic pattern than the current one, so user can add their own limit/requests, e.g.:
```
#option('resourceLimit-hthor-memory', ...);
#option('resourceRequest-hthor-cpu', ...);
```
i.e. so that the postfix is used as the resources sub-property.